lib/extractStyles.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 
 var CSSDisplayNames = require('./CSSDisplayNames');
-var CSSPropertyOperations = require('react/lib/CSSPropertyOperations');
+var explodePseudoStyles = require('./explodePseudoStyles');
```

Comment:
ultra nit: prefer to follow the Smyte styleguide of CamelCase requires first, then newline, then lowercased requires, with each block in alphabetical order